client/src/routes/WorkspaceRoute/Workspace.css

```diff
@@ -34,6 +35,8 @@ body {
 
 .wrapper__column:nth-child(2) {
   width: 60vw;
+  border-top: 6px solid #02feff;
+  border-bottom: 6px solid #02feff;

```

Comment:
If `6px solid #02feff` appears multiple times (more than like 3 times), you can make it like a variable by writing:
```
:root {
--main-border: 6px solid #02feff;
}
```
And then write smth like `border-top: var(--main-border);` instead.
Ignore this if this only appears twice here in this file.

Reply:
this only appears twice, but thanks for the suggestion!